src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentGeneratedOutputTracker.cs

```diff
@@ -16,6 +18,7 @@ internal class DocumentGeneratedOutputTracker
         private Task<RazorCodeDocument> _task;
         
         private IReadOnlyList<TagHelperDescriptor> _tagHelpers;
+        private IReadOnlyList<ImportItem> _imports;
```

Comment:
I'd expect this to have its own 'tracker'. It might be a little vague, but the pattern is that the 'tracker' is the holder of some optional state. 

If this isn't clear, or isn't useful then maybe this is a good sign that the design as it is doesn't add much value.

Reply:
I'm not sure that this detail is important right now, so feel free to ignore.